reference/clinic.v1.yaml

```diff
@@ -1010,6 +1010,28 @@ paths:
       description: 'Delete a Site'
       tags:
         - Clinics
+  '/v1/clinics/{clinicId}/sites/{siteId}/merge':
+    parameters:
+      - $ref: '#/components/parameters/clinicId'
+      - $ref: '#/components/parameters/siteId'
+    post:
+      summary: Merge two sites
+      operationId: MergeSite
+      responses:
+        '200':
+          description: The merged clinic site
+          content:
+            application/json:
+              schema:
+                $ref: ./clinic/models/site.v1.yaml
+      description: Merge a site, identified in the request body, into the site identified in the URL.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/site.v1'
```

Comment:
The payload schema isn't correct. The validator will expect a site name to present in the payload (because it's required), but only the id is required and used by the underlying implementation.